### ISSUE TITLE
[CLI] Shouldn't `--wit-dir` also change the place where we look-up for `wkg.(toml|lock)`?

### ISSUE THREAD

**Issue by raskyld** [bytecodealliance/wasm-pkg-tools#137]
Hello!

I have a repo with multiple WIT packages, I use [this option](https://github.com/bytecodealliance/wasm-pkg-tools/blob/main/crates/wkg/src/wit.rs#L46), so I can build my different package.

I noticed that we use the default `(Lock|Config)::load()`, but should we instead use [this function](https://github.com/bytecodealliance/wasm-pkg-tools/blob/main/crates/wasm-pkg-core/src/config.rs#L32) to load from the dir passed to `--wit-dir`?

If so, I would be happy to contribute back a quick MR! ❤️

**Comment by thomastaylor312** [comment]
I've gone back and forth on what the best behavior and location would be here. Other `.lock` files are almost always at the top level of a project, but also, `wit` is kinda half way between a vendor/node_modules dir and configuration. The other use case I see here is where you do want things at the top level (current behavior), but your wit directory is not in the standard location. 

@joonas @calvinrp curious if you have any thoughts here

**Comment by raskyld** [comment]
If that may bring any perspective, in my specific use-case, I have different packages in a repository.

What you describe "things at the root" sounds more like the equivalent of "workspaces" in Cargo and go modules.

**Comment by thomastaylor312** [comment]
Yeah for sure @raskyld! That is what I am trying to tease out. I've seen people use it both ways. Just trying to figure out where the right place would be to store it and why.
